README.rst

```diff
@@ -5,6 +5,25 @@ A sunpy-affiliated package for solar instrument-specific tools.
     :target: http://www.sunpy.org
     :alt: Powered by SunPy Badge
 
+What is sunkit-instruments?
+---------------------------
+
+sunkit-instruments is a sunpy-affiliated package for solar instrument-specific tools.
+Its purpose is not to be a repository for all tools for all instruments.
+Instead it is intended to perform three main roles:
+
+1. Hold instrument tools that are so few they do not justify their own package;
+2. Hold tools for instruments with no instrument team or the instrument team does not currently support solar applications;
```

Comment:
maybe give an example for `instrument team does not currently support solar applications` and have something along the line of e.g. Fermi? Or explicitly that for an instrument is not solar focused?

Reply:
Not sure, we want this.  What if that changes?  It makes this document an extra thing we have to remember to maintain.  I would suggest that the fact that fermi is in the package IS the example.